### ISSUE TITLE
Version 2.25.1 - Request for GUI fixes

### ISSUE THREAD

**Issue by bovirus** [murrty/youtube-dl-gui#58]
@murrty 

Some issues about v. 2.25.1 (the version should be 2.25.1 - not 2.251)

![youtube-dl-gui-2 1 5 1-01](https://user-images.githubusercontent.com/1262554/97174205-116dba80-1792-11eb-815b-8c13ac16728b.jpg)
Please change defualt main window width (at first run).
I know that user cam set a size and save but my request is about to set a default first run value for main window width.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-02](https://user-images.githubusercontent.com/1262554/97174208-12065100-1792-11eb-862a-f577e3eb2d9f.jpg)
Text untranslated.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-03](https://user-images.githubusercontent.com/1262554/97174210-12065100-1792-11eb-9e4b-45346e8cc689.jpg)
Move these items few down to have mroe space vs second box.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-04](https://user-images.githubusercontent.com/1262554/97174211-129ee780-1792-11eb-882d-12dd80b3ab97.jpg)
"Sound" is not translated
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-05](https://user-images.githubusercontent.com/1262554/97174212-129ee780-1792-11eb-805d-7fae16b3251b.jpg)
Text strings untranslated.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-06](https://user-images.githubusercontent.com/1262554/97174213-129ee780-1792-11eb-815f-9973e3f95210.jpg)
Text strings untranslated.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-07](https://user-images.githubusercontent.com/1262554/97174216-13377e00-1792-11eb-9234-e5c1cd6aa6d2.jpg)
Text strings untranslated.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-08](https://user-images.githubusercontent.com/1262554/97174217-13377e00-1792-11eb-9065-229826bd65b3.jpg)
Click on this link point to a not existent web page.
---------------------------------------------------------------------------
![youtube-dl-gui-2 1 5 1-09](https://user-images.githubusercontent.com/1262554/97174218-13377e00-1792-11eb-9844-8aeec1c473ed.jpg)
This and ny other drop down file list is untranslated.
If you can add this item names (realted not only this window but at all windows) in translation strings.
---------------------------------------------------------------------------

**Comment by Dr-Flay** [comment]
Hopefully the dev will fix the supported sites list on his own page, as that is also not working
https://yt-dl.org/supportedsites.html
However until it is fixed you could point to a rehost of the list here, or use the web archive last copy
https://web.archive.org/web/20201004065152/http://ytdl-org.github.io/youtube-dl/supportedsites.html

**Comment by murrty** [comment]
The initial form width isn't going to change. I added the option for people, but it's a design choice I made to go with the minimal size of the program windows to keep it from taking up large, unnecessary areas of monitors. People can resize it if they want, but ultimately it's staying as is.

The supported link became unavailable when the original project was removed from Github. Currently, people can use the archive link provided by Dr-Flay above. I'll just change it to that for now until another, up-to-date list is available.

The formats for the tools/conversions aren't going to change right now either. Only real reason is I just don't really want to right now.
